content/metadata.yaml

```diff
@@ -771,12 +771,12 @@ author_info:
 #   twitter: null
 #   affiliations:
 #   - Northeastern University
-# - name: Rashika Ramola
-#   orcid: null
-#   github: null
-#   twitter: null
-#   affiliations:
-#   - Northeastern University
+- name: Rashika Ramola
+  orcid: null
```

Comment:
@dhimmel what is the desired behavior when these author attributes are defined but set to `null`?  Currently, they are displayed as `None`.  Not showing them would look cleaner.

![image](https://user-images.githubusercontent.com/3532898/50766871-72847100-1240-11e9-9222-ff393e558f92.png)

Do we want to change `00.front-matter.md` in manubot-rootstock?  Or check for it in `manubot.process.util`?

Reply:
I agree the desired behavior would be to omit a field if null. 

> Do we want to change 00.front-matter.md in manubot-rootstock? Or check for it in manubot.process.util?

 Yes, I'll try to remember to look into implementing this.